src/sentry/mediators/sentry_apps/internal_creator.py

```diff
@@ -34,29 +34,30 @@ def call(self):
         self.sentry_app.verify_install = False
         self.sentry_app.save()
 
-        self._create_access_token()
         self._install()
+        self._create_access_token()
 
         return self.sentry_app
 
     def _create_access_token(self):
-        self.api_token = ApiToken.objects.create(
-            user=self.sentry_app.proxy_user,
-            application_id=self.sentry_app.application.id,
-            scope_list=self.sentry_app.scope_list,
-            expires_at=None,
-        )
+        data = {
+            'sentry_app_installation': self.install,
+            'user': self.user
+        }
+
+        sentry_app_installation_token = SentryAppInstallationTokenCreator.run(
+            request=self.request, **data)
+        self.install.api_token = sentry_app_installation_token.api_token
```

Comment:
Do we want to keep adding the api_token to the install now that we have the join table?

Reply:
@MeredithAnya For now, yes. I don't want to make any breaking changes in this PR.